src/RestrictedPython/compile.py

```diff
@@ -19,33 +21,40 @@
 
 
 def _compile_restricted_mode(
-        source,
-        filename='<string>',
-        mode="exec",
-        flags=0,
-        dont_inherit=False,
-        policy=RestrictingNodeTransformer):
-
+    source,
+    filename='<string>',
+    mode="exec",
+    flags=0,
+    dont_inherit=False,
+    policy=RestrictingNodeTransformer,
+):
```

Comment:
This is not readable and not PEP-8 conform, see https://www.python.org/dev/peps/pep-0008/#indentation